src/app/pages/Designathons/index.js

```diff
@@ -1,17 +1,21 @@
-import { Route, Routes } from 'react-router-dom';
-import Designathon22 from './Designathon22';
-import Designathon23 from './Designathon23';
-import Home from './Home';
+import { Route, Routes } from "react-router-dom";
+import Designathon22 from "./Designathon22";
+import Designathon23 from "./Designathon23";
+import Home from "./Home";
+import Designathon24 from "./Designathon24/Designathon24";
 
 const Designathon = () => {
 	return (
 		<Routes>
 			<Route index element={<Home />} />
-			<Route path='22' element={<Designathon22 />} />
-			<Route path='impact' element={<Designathon22 />} />
+			<Route path="22" element={<Designathon22 />} />
+			<Route path="impact" element={<Designathon22 />} />
 
-			<Route path='23' element={<Designathon23 />} />
-			<Route path='you-belong-here' element={<Designathon23 />} />
+			<Route path="23" element={<Designathon23 />} />
+			<Route path="you-belong-here" element={<Designathon23 />} />
+
+			<Route path="24" element={<Designathon24 />} />
+			<Route path="true-to-you" element={<Designathon24 />} />
```

Comment:
Thought: Could we temporarily disable this route so the page isn't actually publicly accessible until we are ready to release?

Reply:
I was gonna say, let's introduce a route but we'll call it /beta/24 or something so that we can easily show our stakeholders since we don't have a beta/staging deployment